gen-compdocs/generators/doc.go

```diff
@@ -122,35 +127,31 @@ func GenMarkdownCustom(cmd *cobra.Command, w io.Writer, linkHandler func(string)
 				return err
 			}
 		}
-
-	}	
-
+	}
 
 	/* Examples */
 	if len(cmd.Example) > 0 {
-	if with_title {
```

Comment:
these indent fixes are unrelated to the main change?

Reply:
Yes. The indentation problem hurts the readability when I tried to find out the location to insert this new logic, so I fixed them together.

Reply:
Thanks for fixing the indentation.